near-sdk/src/environment/env.rs

```diff
@@ -948,6 +1721,19 @@ pub fn storage_read(key: &[u8]) -> Option<Vec<u8>> {
 }
 /// Removes the value stored under the given key.
 /// If key-value existed returns `true`, otherwise `false`.
+///
+/// # Use cases
+/// Storage functions are typically used to upgrade/migrate a contract state, preventing errors like `Cannot deserialize the contract state` after rolling out the breaking changes to the network.
+/// For practical examples, see different implementations in [this repository](https://github.com/near-examples/update-migrate-rust).
```

Comment:
this call isn't used in the referenced repo, and also in a pending pr https://github.com/near-examples/update-migrate-rust/pull/2 
won't request change to remove this link, hoping that this and some of other suggestions will be adressed in some of subsequent prs